plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/glue/GlueMetastoreModule.java

```diff
@@ -59,6 +57,17 @@ protected void setup(Binder binder)
                 .annotatedWith(ForRecordingHiveMetastore.class)
                 .to(GlueHiveMetastore.class)
                 .in(Scopes.SINGLETON);
+
+        binder.bind(HiveMetastore.class)
+                .annotatedWith(ForGlueColumnStatisticsRead.class)
+                .to(GlueHiveMetastore.class)
+                .in(Scopes.SINGLETON);
+
+        binder.bind(HiveMetastore.class)
+                .annotatedWith(ForGlueColumnStatisticsWrite.class)
+                .to(GlueHiveMetastore.class)
+                .in(Scopes.SINGLETON);
```

Comment:
those are not used anywhere.

Reply:
Used in GlueHiveMetastore's constructor

Reply:
The `Executors` (which are bound below) are used in `GlueHiveMetastore`. This one is adding binging for `HiveMetastore` (annotated with `ForGlueColumnStatisticsRead` and `ForGlueColumnStatisticsWrite`).